bin/license-json-converter.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const minimist = require('minimist');
+const _ = require('lodash');
+const chalk = require('chalk');
+const semver = require('semver');
+
+const argv = minimist(process.argv);
+const { XLSXBuilder } = require('../lib/builders');
+
+if (argv.help) {
+  console.log('Usage:');
+  console.log('license-json-consolidate [arguments]');
```

Comment:
Presumably this should be `license-json-converter`, maybe need some descriptions in here now that we have a few scripts.